packages/relay/src/lib/config/preconfiguredSpendingPlans.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import fs from 'fs';
+import findConfig from 'find-config';
+import { SpendingPlanConfig } from '../types/spendingPlanConfig';
+import { HbarSpendingPlanRepository } from '../db/repositories/hbarLimiter/hbarSpendingPlanRepository';
+import { EthAddressHbarSpendingPlanRepository } from '../db/repositories/hbarLimiter/ethAddressHbarSpendingPlanRepository';
+import { IPAddressHbarSpendingPlanRepository } from '../db/repositories/hbarLimiter/ipAddressHbarSpendingPlanRepository';
+import { RequestDetails } from '../types';
+import { Logger } from 'pino';
+
+const PRE_CONFIGURED_SPENDING_PLANS_TTL = -1; // Never expire
+const SPENDING_PLANS_CONFIG_FILE = 'spendingPlansConfig.json';
+
+/**
+ * Loads the pre-configured spending plans from a JSON file.
+ * @returns {SpendingPlanConfig[]} An array of spending plan configurations.
+ * @throws {Error} If the configuration file is not found or cannot be read.
+ */
+const loadSpendingPlansConfig = (): SpendingPlanConfig[] => {
+  const path = findConfig(SPENDING_PLANS_CONFIG_FILE);
```

Comment:
Should we allow the operator to configure where to place the spending plans file on their file system?  They may want special protections on it.  Pass in the path from an environment variable?
```
const loadSpendingPlansConfig = (configPath?: string): SpendingPlanConfig[] => {
  const path = configPath || findConfig(SPENDING_PLANS_CONFIG_FILE);
  ```

Reply:
Done, made the filename configurable through `HBAR_SPENDING_PLANS_CONFIG_FILE` env variable